CMakeLists.txt

```diff
@@ -599,6 +599,10 @@ CONFIGURE_FILE(
 	IMMEDIATE @ONLY
 )
 
+# Test setup
+ENABLE_TESTING()
+LIST(APPEND CMAKE_CTEST_ARGUMENTS "--verbose")
```

Comment:
In non-verbose mode I imagine there are no details printed for failed tests?

Can we make this a parameter to `make test`? Most of the time we won't care for the wall of text, just that the tests pass. Then we could do something like this in CI: `make test || make test --verbose --rerun-failed`

Reply:
I justed want to see whether my tests are registered properly (and not simply went missing) because otherwise no (new) tests also means green here.

There is also something like `--output-on-failure` and then CMake refrains from suppressing output of failure cases by running once again.